RecoLocalCalo/HGCalRecProducers/python/hgcalLayerClusters_cff.py

```diff
@@ -26,10 +29,11 @@
     nHitsTime = cms.uint32(3),
     plugin = dict(
         dEdXweights = dEdX.weightsNose,
+        maxNumberOfThickIndices = cms.uint32(3),
         fcPerMip = HGCalUncalibRecHit.HGCHFNoseConfig.fCPerMIP,
         thicknessCorrection = HGCalRecHit.thicknessNoseCorrection,
         fcPerEle = fC_per_ele,
-        noises = dict(refToPSet_ = cms.string('HGCAL_noises')),
+        noises = cms.vdouble(HGCAL_noises.values),
```

Comment:
```suggestion
        noises = HGCAL_noises.values.value(),
```
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.